app/views/overview/_service-binding.html

```diff
@@ -1,20 +1,33 @@
-<div class="row service-binding">
-  <div class="col-sm-5 col-md-6">
-    <h3 ng-if="$ctrl.serviceClass">
-      {{$ctrl.serviceClass.externalMetadata.displayName || $ctrl.serviceClass.metadata.name}}
-      <small>{{$ctrl.binding.spec.instanceRef.name}}</small>
-    </h3>
-    <h3 ng-if="!$ctrl.serviceClass">
-      {{$ctrl.binding.spec.instanceRef.name}}
-    </h3>
+<div class="service-binding">
+  <div class="row">
+    <div class="col-sm-5 col-md-6">
+      <h3 ng-if="$ctrl.serviceClass">
+        <span ng-if="$ctrl.serviceClass">
+          {{$ctrl.serviceClass.externalMetadata.displayName || $ctrl.serviceClass.metadata.name}}
```

Comment:
At some point this is worth a `serviceClassDisplayName` filter. I suspect we have this in a lot of places.